src/main/java/com/starrocks/connector/flink/connection/StarRocksJdbcConnectionOptions.java

```diff
@@ -38,12 +38,24 @@ public class StarRocksJdbcConnectionOptions  implements Serializable {
     @Nullable
     protected final String password;
 
+    protected final String fenodes;
```

Comment:
`fenodes` is not used commonly, and do not add it here

Reply:
already delete it